internal/dao/log_item.go

```diff
@@ -65,12 +75,11 @@ func (l *LogItem) Size() int {
 
 // Render returns a log line as string.
 func (l *LogItem) Render(paint string, showTime bool, bb *bytes.Buffer) {
-	index := bytes.Index(l.Bytes, []byte{' '})
-	if showTime && index > 0 {
+	if showTime && l.Time != "" {
 		bb.WriteString("[gray::b]")
-		bb.Write(l.Bytes[:index])
+		bb.Write([]byte(l.Time))
```

Comment:
I think we should keep the Time field as bytes vs string to avoid this allocation.